ElitismHelper.lua

```diff
@@ -445,6 +444,21 @@ SlashCmdList["ELITISMHELPER"] = function(msg,editBox)
 			print(" activeUser: Prints active user")
 			print(" output: Define output channel between default | party | raid | yell | self")
 		end,
+		["threshold"] = function()
```

Comment:
Are you missing the argument to the function?
```suggestion
		["threshold"] = function(argsFunc)
```

Reply:
Yep, good catch there